frontend/packages/kubevirt-plugin/src/k8s/helpers/vm-clone.ts

```diff
@@ -102,15 +102,29 @@ export class VMClone {
         const pvc = pvcLookup[`${this.oldVMNamespace}-${pvcName}`];
 
         if (pvc) {
-          const clonedDVTemplate = new DataVolumeTemplate({
-            name: generateDataVolumeName(name, volume.name),
-            pvcSourceName: pvcName,
-            pvcSourceNamespace: this.oldVMNamespace,
-            accessModes: getPvcAccessModes(pvc),
-            volumeMode: getPvcVolumeMode(pvc),
-            size: getPvcStorageSize(pvc),
-            storageClassName: getPvcStorageClassName(pvc),
-          }).build();
+          const clonedDVTemplate: V1DataVolumeTemplateSpec = {
```

Comment:
can we rather use a DataVolumeWrapper here for easier use?

We could add a special method there similar to asResource for compatibility with V1DataVolumeTemplateSpec